docs/en/administration/management/compaction.md

```diff
@@ -171,12 +169,23 @@ ADMIN SET FRONTEND CONFIG ("lake_compaction_max_tasks" = "-1");
 - Description: The maximum number of concurrent Compaction tasks allowed in a shared-data cluster. Setting this item to `-1` indicates calculating the concurrent task number in an adaptive manner, that is, the number of surviving CN nodes multiplied by 16. Setting this value to `0` will disable compaction.
 - Introduced in: v3.1.0
 
+```SQL
+ADMIN SET FRONTEND CONFIG ("lake_compaction_disable_tables" = "11111;22222");
+```
+
+##### lake_compaction_disable_tables
+
+- Default：""
+- Type：String
+- Unit：-
+- Is mutable：Yes
+- Description：Disable compaction for certain tables, does not affect compaction that has started. Value is Table Id, seperated by ';'.
```

Comment:
```suggestion
- Description：Disable compaction for certain tables. This will not affect compaction that has started. The value of this item is table ID. Multiple values are separated by ';'.
```